pkgs/development/r-modules/default.nix

```diff
@@ -309,7 +309,6 @@ let
   };
 
   packagesWithNativeBuildInputs = {
-    arrow = [ pkgs.pkg-config pkgs.arrow-cpp ];
     adimpro = [ pkgs.imagemagick ];
     animation = [ pkgs.which ];
     audio = [ pkgs.portaudio ];
```

Comment:
```suggestion
    arrow = with pkgs; [ pkg-config cmake  ] ++ lib.optionals stdenv.isDarwin [ intltool ];
    audio = [ pkgs.portaudio ];
```

Reply:
Actually, these three packages (pkg-config, cmake and likely intltool) belong under packagesWithNativeBuildInputs, but arrow does not. So if you don't mind, it would be good to place these back here, and the modified arrow should be a regular buildInput.